### ISSUE TITLE
Scanning music library fails on 10.6

### ISSUE THREAD

**Issue by theg00s3** [jellyfin/jellyfin#3606]
**Describe the bug**
When I try to scan my music library, the scanner fails

**System (please complete the following information):**
 - OS: Ubuntu 20.04
 - Jellyfin Version: 10.6
 - Storage: local

**To Reproduce**
1. Go to Dashboard -> Libraries
2. Click on the three dots -> scan library -> Scan for new and updated files
4. Check log

**Expected behavior**
The scan to complete successfully

**Logs**
````
[2020-07-20 16:03:41.981 +10:00] [INF] OnRefreshStart "6d30fb6dd8a3c0cccce5e6fa1bdb784c"
[2020-07-20 16:03:57.654 +10:00] [INF] OnRefreshComplete "6d30fb6dd8a3c0cccce5e6fa1bdb784c"
[2020-07-20 16:03:57.691 +10:00] [ERR] Error refreshing item
MediaBrowser.Model.Net.HttpException: Origin Connection Time-out
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.EnsureSuccessStatusCode(HttpResponseMessage response, HttpRequestOptions options)
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsyncInternal(HttpRequestOptions options, HttpMethod httpMethod)
   at Emby.Server.Implementations.HttpClientManager.HttpClientManager.SendAsync(HttpRequestOptions options, HttpMethod httpMethod)
   at MediaBrowser.Providers.Manager.ProviderManager.SaveImage(BaseItem item, String url, ImageType type, Nullable`1 imageIndex, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Library.LibraryManager.ConvertImageToLocal(BaseItem item, ItemImageInfo image, Int32 imageIndex)
   at Emby.Server.Implementations.Library.LibraryManager.UpdateImages(BaseItem item, Boolean forceUpdate)
   at MediaBrowser.Controller.Entities.Folder.ValidateChildrenInternal2(IProgress`1 progress, CancellationToken cancellationToken, Boolean recursive, Boolean refreshChildMetadata, MetadataRefreshOptions refreshOptions, IDirectoryService directoryService)
   at MediaBrowser.Controller.Entities.Folder.ValidateChildrenInternal(IProgress`1 progress, CancellationToken cancellationToken, Boolean recursive, Boolean refreshChildMetadata, MetadataRefreshOptions refreshOptions, IDirectoryService directoryService)
   at MediaBrowser.Providers.Manager.ProviderManager.RefreshCollectionFolderChildren(MetadataRefreshOptions options, CollectionFolder collectionFolder, CancellationToken cancellationToken)
   at MediaBrowser.Providers.Manager.ProviderManager.RefreshItem(BaseItem item, MetadataRefreshOptions options, CancellationToken cancellationToken)
   at MediaBrowser.Providers.Manager.ProviderManager.StartProcessingRefreshQueue()

**Comment by theg00s3** [comment]
It's cleared itself up somehow. I haven't restarted the server, or touched the config, but it's working now

**Comment by CountMurphy** [comment]
Odd. I'm having the same issue. I wonder if the issue is on the server jellyfin is calling to ......

**Comment by theg00s3** [comment]
still active

**Comment by cvium** [comment]
Fixed in 10.7.2
